lib/parse-message.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+module.exports = exports = {};
+
+exports.usernameCommand = function (message, socket) {
+  socket.username = message.split(' ').slice(1).join(' ').trim();
+  socket.write(`Your username is now ${socket.username}`);
+};
+
+exports.dmCommand = function (message, array, socket) {
+  let toUsername = message.split(' ')[1];
+  let content = message.split (' ').slice(2).join(' ').trim();
+  array.forEach(s => {
+    if (s.username === toUsername)
+      s.write(`${socket.username}${content}`);
```

Comment:
Format this better with a ``:`` in between username and content, i.e.:

```
`${socket.username}: ${content}`
```

When I dm another user, their message gets displayed as one jumbled up word.